torch_em/model/unetr.py

```diff
@@ -41,7 +41,18 @@ def _load_encoder_from_checkpoint(self, backbone, encoder, checkpoint):
                     encoder_state = torch.load(checkpoint)
 
             elif backbone == "mae":
-                encoder_state = torch.load(checkpoint)
+                # vit initialization hints from:
+                #     - https://github.com/facebookresearch/mae/blob/main/main_finetune.py#L233-L242
+                encoder_state = torch.load(checkpoint)["model"]
+                encoder_state = OrderedDict({
+                    k: v for k, v in encoder_state.items()
+                    if (k != "mask_token" and not k.startswith("decoder"))
+                })
+
+                # let's remove the `head` from our current encoder (as the MAE pretrained don't expect it)
+                current_encoder_state = self.encoder.state_dict()
+                if "head.weight" and "head.bias" in current_encoder_state:
```

Comment:
This only checks if `"head.bias" in current_encoder_state`. `"head.weight"` is evaluated as an independent expression and will always evaluate to True.

Reply:
Ah my bad. I just overlooked it. Thanks for pointing. It's fixed now!